tox.ini

```diff
@@ -0,0 +1,51 @@
+[pytest]
+addopts =
+    --tb=short
+    -vv
+    --strict-markers
+    --cov-report term-missing
+    --cov-fail-under 100
+    -p no:warnings
+testpaths = common_migration/tests
+
+[flake8]
+
+exclude = ve,migration_files
+; Set the hard line length limit to 90 characters.
+max-line-length = 90
+; W503 - Allow binary operators to start a line
+; E402 - Allow imports to be at the start of the file
+; FI1* - Errors for missing `__future__` imports.
+; FI90 - Other __future__ imports.
+ignore =
+    W503,
+    E402,
+    FI10,
+    FI11,
+    FI12,
+    FI13,
+    FI14,
+    FI15,
+    FI16,
+    FI17,
+    FI90
+
+[tox]
+envlist =
+    flake8,
+    pyright,
+    pytest
+
+[testenv]
+setenv = PYTHONPATH=""
+deps = -rdev-requirements.txt
```

Comment:
Are you missing a space here `-r dev-requirements.txt`?

Reply:
I think it doesn't work with a space included.